### DISCUSSION TITLE
What's the meaning in benchmark result?

### DISCUSSION THREAD

**Discussion by gakki2019** [zlib-ng/zlib-ng#1817] [Q&A]
![image](https://github.com/user-attachments/assets/f565dbd2-2220-4b61-9ee6-6e05ac998cdd)
compress_bench/compress_bench/32768         687253 ns       686892 ns         1025

32768 means 32768 bytes take 687253 ns,  almost 45.5MBytes/s for compress,  Is my understanding correct?

**Comment by nmoinvaz** [comment]
It is the size of the buffer to compress for the test.

**Comment by KungFuJesus** [comment]
Bear in mind that's also a pretty compressible set of bytes. Compression speed is going to vary highly by the content and compressibility.
